_docs/sdk/ios.md

```diff
@@ -25,7 +25,7 @@ When compiling your app with Smartlook for debugging, you may encounter list of
 
 ## WiFi / mobile connection / offline
 
-By default, we are uploading sessions **only of WiFi** connection. This settings can could be changed in your app dashboard.
+By default, we are uploading sessions **only of WiFi** connection. This setting can be changed in your apps dashboard.
```

Comment:
I would change this to `in your Smartlook dashboard`.

Reply:
Changed.